.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -39,21 +39,10 @@ steps:
         MAPPED_ADO_PAT: $(ADO_PAT)
   displayName: 'az devops login'
 - bash: |
-        echo "PR for Image Bumping, Official Branch Cutting"
-        /bin/bash vhdbuilder/scripts/automate_version_bump.sh $(IMAGE_VERSION) $MAPPED_GITHUB_PAT "${BUILD_ID[@]}"
+        echo "Bumping windows VHD base image version"
```

Comment:
Will it block the current pipeline?

Reply:
Meaning? When running the bump, only the checkbox "Image Bump + Branch Cutting" should be marked on the portal which only creates the pull request e.g. https://github.com/Azure/AgentBaker/pull/3435/files that includes the windows base image version change.

Since `ImageBump` will only be triggered once, I don't think it will block.

Reply:
what is the removed lines used for? Is it used by Linux pipeline?

Reply:
It is used by the Linux pipeline to bump the version given the UI portal pipeline variable `BUILD_ID` and 'IMAGE_VERSION`. Line 43 is just the windows version.